### DISCUSSION TITLE
Error trying Build the extension

### DISCUSSION THREAD

**Discussion by Ahmedchri** [aiortc/aiortc#999] [Q&A]
Hello I'm trying to install RTCBot and I'm following the official documentation in rtcbot website https://rtcbot.readthedocs.io/en/latest/installing.html in windows, I'm in the installation of airtoc step and to be exact to the build of extension after extracting libopus and ibvpx.
I got this error that I don't understand after executing this command
py setup.py build_ext --include-dirs=./include --library-dirs=./lib/x64:
C:\...Python\Python312\Lib\site-packages\setuptools\_distutils\dist.py:265: UserWarning: Unknown distribution option: 'cffi_modules'
  warnings.warn(msg)
running build_ext
thank you for your response!

**Comment by jlaine** [comment]
I don't understand why rtcbot recommends building `aiortc` from source.. especially on Windows!

I'd advise you to just `pip install aiortc`

**Comment by hilookas** [comment]
try: pip install --upgrade pip
